### PR TITLE
test: two small tweaks to the tests

### PR DESCRIPTION
**Pull request by nedbat**
I keep finding small improvements, I hope the stream of PRs isn't a nuisance.

**Merge checklist:**
Check off if complete *or* not applicable:
- [ ] Changelog record added
- [ ] Documentation updated (not only docstrings)
- [ ] Fixup commits are squashed away
- [ ] Unit tests added/updated
- [ ] Manual testing instructions provided
- [ ] Noted any: Concerns, dependencies, deadlines, tickets

### CODE REVIEW

File: tests/test_cookiecutter_python_library.py [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "author_email": "cookie@monster.org",
     "author_name": "Cookie Monster",
     "library_name": "cookie_lover",
-    "github_org": "bakery_org",
+    "github_org": "openedx",
```
Comment (timmc-edx) [inline tests/test_cookiecutter_python_library.py:35]: I infer that you've made this change to test the new branch, and that the other cookiecutter tests use non-openedx orgs?

**Comment by timmc-edx** [comment]
What prompted the changes to the hardcoding test?

File: tests/test_cookiecutter_python_library.py (hunk 1 above)
Comment (nedbat) [reply tests/test_cookiecutter_python_library.py:35]: Yes, this was to have a mixture of organizations across all of the tests.

**Comment by nedbat** [comment]
@timmc-edx Are you still waiting on something from me?

**Comment by timmc-edx** [comment]
Yeah, not sure if you saw this comment:

> What prompted the changes to the hardcoding test?

I'm wondering what the failure mode was that prompted this (whether observed or hypothetical).

**Comment by nedbat** [comment]
If you mean counting the right and wrong lines, I was curious about how many lines there were, and figured there should be something more than "no bad lines."  There should also be some good lines.

**Comment by timmc-edx** [comment]
Got it, makes sense.

**Review by timmc-edx — APPROVED** [review]

**Pull request merged**